### DISCUSSION TITLE
Como adiciona?

### DISCUSSION THREAD

**Discussion by abacaxiatomico** [geraldohomero/best-foss-alternatives#42] [Q&A] [answered]
Como adicionar um na lista?

**Comment by geraldohomero** [comment, marked as answer]
Sinta-se à vontade para [contribuir](https://github.com/geraldohomero/best-foss-alternatives/blob/main/CONTRIBUTING.md) para a lista!

## Faça um Fork do projeto

Add o nome e o link do projeto (sempre em ordem alfabética)

Use markdown

info = NOS, android, ios, paid, online

- `[<nome>](<link>) - <info>`
```bash
## Browsers
- [Brave](https://brave.com)
- [DuckDuckGo](https://duckduckgo.com/)
- [Firefox](https://mozilla.org)
- [Floorp](https://floorp.app/)
- [LibreWolf](https://librewolf.net/)
- [Mullvad Browser](https://mullvad.net/en/download/browser)
- [Tor](https://www.torproject.org)
```
Faça um pull request

## Ou vc pode simplesmente abrir uma [issue](https://github.com/geraldohomero/best-foss-alternatives/issues) com o software que vc quiser que verifico e adiciono para vc!
